### ISSUE TITLE
FIle list - sort page by filename or date?

### ISSUE THREAD

**Issue by pejacoby** [JosephHewitt/wardriver_rev3#141]
As more files are collected on the SD card, the order in which they are presented in the GUI becomes jumbled.  Is it possible to sort the file list by filename or by date?  Putting them filename by reverse order would put the latest files at the top (highest numbers).
Sorting by date might not be as reliable as files initiated without a network connection may get 1970s dates if the correct time has not been set.

**Comment by JosephHewitt** [comment]
Agreed. The best solution for this would be to sort the files using Javascript in the browser.

I'm likely to proceed with the new webapp idea discussed here: https://github.com/JosephHewitt/wardriver_rev3/discussions/119 so this would be a good feature to have implemented there.

**Comment by pejacoby** [comment]
I've made a first stab at adding sorting to the web interface and it works pretty well.  Injecting the javascript into the page structure isn't super pretty, but it works.  I have a bit more testing to do but will look at pushing things into my fork for others to see in the coming week.
